### ISSUE TITLE
PlatformDispatcher.accessibilityFeatures complete with properties for 'Button Shapes' and 'On/Off Labels'

### ISSUE THREAD

**Issue by fabianbrandscheid** [flutter/flutter#86927]
Hi,

I use the PlatformDispatcher to query some accessibility features at app startup. I noticed that there is currently no possibility to query the iOS specific settings 'Button Shapes' and 'On/Off Labels'. Could the AccessibilityFeatures of the PlatformDispatcher be extended by these two options?

Thank you very much and best regards

**Comment by zanderso** [comment]
@chunhtai any thoughts about this?

**Comment by chunhtai** [comment]
Yes this is possible since iOS does provide the API to query this data, we can hook them to the AccessibilityFeatures

https://developer.apple.com/documentation/uikit/3618943-uiaccessibilitybuttonshapesenabl?language=objc
https://developer.apple.com/documentation/uikit/3131973-uiaccessibilityisonoffswitchlabe?language=objc
